src/XMonad/Config.hs

```diff
@@ -58,14 +60,22 @@ import Graphics.X11.Xlib.Extras
 workspaces :: [WorkspaceId]
 workspaces = map show [1 .. 9 :: Int]
 
--- | modMask lets you specify which modkey you want to use. The default
+-- | 'modMask' lets you specify which modkey you want to use. The default
 -- is mod1Mask ("left alt").  You may also consider using mod3Mask
 -- ("right alt"), which does not conflict with emacs keybindings. The
 -- "windows key" is usually mod4Mask.
 --
 defaultModMask :: KeyMask
 defaultModMask = mod1Mask
 
+-- | 'stripModMask' lets you specify which modifiers are irrelevant for key
+-- bindings. The default is Num Lock and Caps Lock. You will need to override
+-- this if you wish to only strip Caps Lock; e.g., if you need to bind numpad keys
+-- but only when Num Lock is off (or on). Another use case is adding
+-- 'mod5Mask' to the list of stripped/irrelevant modifiers.
+defaultStripModMask :: X [KeyMask]
+defaultStripModMask = gets numberlockMask <&> \numLockMask -> [lockMask, numLockMask]
```

Comment:
Is `gets ((:[lockMask]) . numberlockMask)` better? I don't know. :-)

Reply:
I don't think one is more obscure than the other, so feel free to pick the one you like best :)